tests/posix/common/src/clock.c

```diff
@@ -74,6 +72,8 @@ void test_posix_realtime(void)
 	zassert_equal(rts.tv_sec, mts.tv_sec, "Seconds not equal");
 	zassert_equal(rts.tv_nsec, mts.tv_nsec, "Nanoseconds not equal");
 
+	printk("POSIX clock set APIs\n");
```

Comment:
why is this being moved to here? Wasn't this just moved the other way round in 2b4457347d26ca1e5e79af8f460315111a604421?

Reply:
My mistake! leftover from debug. Thank you!